.travis.yml

```diff
@@ -14,6 +14,7 @@ git:
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -y python-ply python-git libelf-dev codespell fakeroot gcc g++ gcc-aarch64-linux-gnu
+  - sudo apt-get install -y clang-format clang-tidy clang-tools clang clangd
```

Comment:
I think we may need to update to 20.04 focal before try to install below packages.
PS: to work with Linux we may need to use clang-10
```
No apt package "clangd", but there is a snap with that name.
Try "snap install clangd"
E: Unable to locate package clangd
The command "sudo apt-get install -y clang-format clang-tidy clang-tools clang clangd" failed and exited with 100 during .
```